From 7a252061b51b03462a3c818d515124cc5f23fb14 Mon Sep 17 00:00:00 2001
Message-Id: <patch-1.5-7a252061b51-20210928T130032Z-avarab@gmail.com>
In-Reply-To: <cover-0.5-00000000000-20210928T130032Z-avarab@gmail.com>
References: <cover-0.5-00000000000-20210928T130032Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Mon, 27 Sep 2021 01:55:27 +0200
Subject: [PATCH 1/5] refs.[ch]: remove unused ref_storage_backend_exists()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This function was added in 3dce444f178 (refs: add a backend method
structure, 2016-09-04), but has never been used by anything. The only
caller that might care uses find_ref_storage_backend() directly.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 refs.c | 5 -----
 refs.h | 2 --
 2 files changed, 7 deletions(-)

diff --git a/refs.c b/refs.c
index 8b9f7c3a80a..32349a5585f 100644
--- a/refs.c
+++ b/refs.c
@@ -33,11 +33,6 @@ static struct ref_storage_be *find_ref_storage_backend(const char *name)
 	return NULL;
 }
 
-int ref_storage_backend_exists(const char *name)
-{
-	return find_ref_storage_backend(name) != NULL;
-}
-
 /*
  * How to handle various characters in refnames:
  * 0: An acceptable character for refs
diff --git a/refs.h b/refs.h
index 48970dfc7e0..573b69863d3 100644
--- a/refs.h
+++ b/refs.h
@@ -836,8 +836,6 @@ int reflog_expire(const char *refname, const struct object_id *oid,
 		  reflog_expiry_cleanup_fn cleanup_fn,
 		  void *policy_cb_data);
 
-int ref_storage_backend_exists(const char *name);
-
 struct ref_store *get_main_ref_store(struct repository *r);
 
 /**
-- 
2.33.0.1340.ge9f77250f2b

